deploy/examples/common-second-cluster.yaml

```
@@ -7,145 +7,145 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: $NAMESPACE
+  name: rook-ceph-secondary # namespace:cluster
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-cluster-mgmt
-  namespace: $NAMESPACE
+  namespace: rook-ceph-secondary # namespace:cluster
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: rook-ceph-cluster-mgmt
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-system
-    namespace: rook-ceph
+    namespace: rook-ceph # namespace:operator
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-cmd-reporter
-  namespace: $NAMESPACE
+  namespace: rook-ceph-secondary # namespace:cluster
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: rook-ceph-cmd-reporter
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-cmd-reporter
-    namespace: $NAMESPACE
+    namespace: rook-ceph-secondary # namespace:cluster
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-cmd-reporter-psp
-  namespace: $NAMESPACE
+  namespace: rook-ceph-secondary # namespace:cluster
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: psp:rook
 subjects:
   - kind: ServiceAccount
     name: rook-ceph-cmd-reporter
-    namespace: $NAMESPACE
+    namespace: rook-ceph-secondary # namespace:cluster
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-mgr-system
-  namespace: $NAMESPACE
+  namespace: rook-ceph # namespace:operator
```

This was causing the CI to fail
```suggestion
  namespace: rook-ceph-secondary # namespace:cluster
```

---

@travisn in `common.yaml` we have the following... so I'm wondering why it's different from one file to the other.

```
# Allow the ceph mgr to access resources in the Rook operator namespace necessary for mgr modules
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: rook-ceph-mgr-system
  namespace: rook-ceph # namespace:operator
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: rook-ceph-mgr-system
subjects:
  - kind: ServiceAccount
    name: rook-ceph-mgr
    namespace: rook-ceph # namespace:cluster

```

---

That one looks incorrect to me, it should be `# namespace:cluster`

---

Fixed, I'll create a separate PR for the other one as it touchs the main `common.yaml ` file